UM/Scene/Camera.py

```diff
@@ -30,7 +31,7 @@ def __init__(self, name, parent = None):
 
     ##  Get the projection matrix of this camera.
     def getProjectionMatrix(self):
-        return self._projection_matrix
+        return copy.deepcopy(self._projection_matrix)
```

Comment:
Because that avoids issues where the matrix is modified later on. I ran into this because multiply/preMultiply modify the matrix instead of returning a new copy.